src/arena/CreateSprintTemplate.tsx

```diff
@@ -140,7 +145,7 @@ function CreateSprintTemplate(props: CreateSprintTemplateProps) {
     };
     try {
       await RestAPI.post("pareto", `/templates`, { body });
-      props.history.push("/");
+      navigate("/");
```

Comment:
Weird lol, will take some time to get used to.

Reply:
Yes, I had to read the docs many times to make sure I was doing things right. Im glad they had a path to use v5 and v6 at the same time so I could tell if I was doing something wrong by breaking it somehow.